packages/analytics-js-plugins/src/deviceModeDestinations/utils.ts

```diff
@@ -38,21 +42,14 @@ const isDestinationSDKEvaluated = (
   destSDKIdentifier: string,
   sdkTypeName: string,
   logger?: ILogger,
-): boolean => {
-  try {
-    const scriptIsEvaluated = Boolean(
-      (globalThis as any)[destSDKIdentifier] &&
-        (globalThis as any)[destSDKIdentifier][sdkTypeName] &&
-        (globalThis as any)[destSDKIdentifier][sdkTypeName].prototype &&
-        typeof (globalThis as any)[destSDKIdentifier][sdkTypeName].prototype.constructor !==
-          'undefined',
-    );
-    return scriptIsEvaluated;
-  } catch (e) {
-    logger?.error(`Error while checking if destination SDK is evaluated: ${e}`);
```

Comment:
we need this try catch as IE11 with trow here if initialization did not work or if the third party does not support IE. Please keep the try catch

Reply:
Do you remember what error it was?
Based on the logic we have, error handling seemed redundant as we're validating resources at every step.

Reply:
https://app.bugsnag.com/rudderlabs/sdk-error-reporter/errors/64a54fbe5ff3ab000923a6dc?filters[event.unhandled]=false&filters[event.since]=30d&filters[app.release_stage]=production&filters[error.status]=open&filters[error.assigned_to]=anyone

Reply:
somehow IE 11 still throws on prototype

Reply:
That's because there seems to be a flaw in the logic.
```
typeof window[pluginName][modName].prototype &&
typeof window[pluginName][modName].prototype.constructor !== 'undefined'
```

It is rectified in V3:
```
window[pluginName][modName].prototype &&
typeof window[pluginName][modName].prototype.constructor !== 'undefined'
```

Reply:
Raised a quick PR to fix it in v1.1 as well, https://github.com/rudderlabs/rudder-sdk-js/pull/1204